options/rtdl/generic/main.cpp

```diff
@@ -108,10 +108,12 @@ extern "C" {
 	}
 }
 
-extern "C" [[gnu::alias("dl_debug_state"), gnu::visibility("default")]] void _dl_debug_state();
+#ifndef __clang__
```

Comment:
Why the ifdef for clang?

Reply:
clang does not like the static alias, does dl_debug_state need to be static?

Reply:
What error does Clang print?